src/tools/tidy/src/features.rs

```diff
@@ -91,7 +91,8 @@ pub fn check(path: &Path, bad: &mut bool, quiet: bool) {
 
     super::walk_many(&[&path.join("test/compile-fail"),
                        &path.join("test/compile-fail-fulldeps"),
-                       &path.join("test/parse-fail"),],
+                       &path.join("test/parse-fail"),
+                       &path.join("test/ui"),],
```

Comment:
Btw as my PR  #46532 which I've done in the meantime (and which was motivated by this PR in fact :p) got merged, you won't need this any more if you rebase.